lineman/app/components/thread_page/thread_page.haml

```diff
@@ -52,3 +52,5 @@
 
     %activity_card{discussion: 'threadPage.discussion'}
     %comment_form{comment: 'threadPage.comment'}
+
+  %thread_lintel
```

Comment:
@gdpelican I have moved the lentil out of the navbar as it really belongs in the %main.thread-page div. Then i can inherit all of its styles.
I have it working where it is but it needs to be indented on more position. Unfortunately it stops working for reasons i can't tell.  Do you think you could have a look at this when you get some time?